.env.example

```
@@ -76,7 +76,7 @@ PINECONE_TOP_K=5
 
 # Hugging Face Configuration
 HUGGINGFACE_HUB_TOKEN=your-huggingface-token
-HUGGINGFACE_CHAT_MODEL=microsoft/DialoGPT-medium
+HUGGINGFACE_CHAT_MODEL=bert-base-uncased
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The HUGGINGFACE_CHAT_MODEL key should go before the HUGGINGFACE_HUB_TOKEN key

(UnorderedKey)